owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/security/PassCodeViewModel.kt

```diff
@@ -51,11 +56,20 @@ class PassCodeViewModel(
         return isValid
     }
 
-    private fun loadPinFromOldFormatIfPossible(): String {
-        var pinString = ""
-        for (i in 1..4)
-            pinString += preferencesProvider.getString(PassCodeActivity.PREFERENCE_PASSCODE_D + i, null)
+    fun getNumberOfPassCodeDigits(): Int {
+        val numberOfPassCodeDigits = contextProvider.getInt(R.integer.passcode_digits)
+        return if (numberOfPassCodeDigits >= PassCodeActivity.PASSCODE_MIN_LENGTH) numberOfPassCodeDigits else PassCodeActivity.PASSCODE_MIN_LENGTH
```

Comment:
We could get rid of some verbosity using `maxOf`, what do you think?
```suggestion
        return maxOf(numberOfPassCodeDigits, PassCodeActivity.PASSCODE_MIN_LENGTH)
```

Reply:
Great, I didn't know about that one, thanks for the tip!